druid-shell/src/gtk/dialog.rs

```diff
@@ -48,25 +45,31 @@ pub(crate) fn get_file_dialog_path(
     ty: FileDialogType,
     options: FileDialogOptions,
 ) -> Result<OsString, Error> {
-    let dialog = match ty {
-        FileDialogType::Open => build_open_dialog(window, options),
-        FileDialogType::Save => build_save_dialog(window, options),
+    let (title, action) = match ty {
+        FileDialogType::Open => ("Open File", FileChooserAction::Open),
```

Comment:
Out of scope for this patch, but we will need to figure out an API here that allows for localization; this means threading through the title from `druid`. You could add a `TODO` here, if you wanted.